source/DATASETS.md

```diff
@@ -10,6 +12,16 @@ Note that BIDS session names have no meaning apart from being data acquired in t
 ## Participants
 Six healthy participants (aged 31 to 47 at the time of recruitment in 2018), 3 women (`sub-03`, `sub-04` and `sub-06`) and 3 men (`sub-01`, `sub-02` and `sub-05`) consented to participate in the Courtois Neuromod Project for at least 5 years. Three of the participants reported being native francophone speakers (`sub-01`, `sub-02` and `sub-04`), one as being a native anglophone (`sub-06`) and two as bilingual native speakers (`sub-03` and `sub-05`). All participants reported being in good general health, all had normal hearing and vision for their age, and all were MRI and MEG compatible.
 
+## anat
+
+The anatomical dataset intends to acquire longitudinal anatomical image of the brain and upper spinal cord at an approximate rate of 4 sessions a year.
+
+The MRI sequences are described in more detailed in :ref:`Brain anatomical sequences` :ref:`Spinal cord anatomical sequences`, including pdfs of the Siemens exam cards.
+
+Brain T1w, T2w and DWI were copied from the HCP aging and development protocol for Prisma MRI scanner.
```

Comment:
Maybe add something like "Other sequences were selected and optimized by the Courtois NeuroMod team."